db/options.go

```diff
@@ -76,6 +78,16 @@ func WithNewThreadKey(key thread.Key) NewOption {
 	}
 }
 
+// WithNewLogKey is the public or private key used to write log records.
+// If this is just a public key, the service itself won't be able to create records.
+// In other words, all records must be pre-created and added with AddRecord.
+// If no log key is provided, one will be created internally.
+func WithNewLogKey(key crypto.Key) NewOption {
```

Comment:
Great! For some reason I assumed this wouldn't be desired at the DB level.

Reply:
It actually slipped into the PR, but anyway we have db failures without it.